tests/test_mapperImport.py

```diff
@@ -56,8 +58,8 @@ def checkIO(self, butler, bbox, ccd):
         butler.put(bbox, "x", ccd=ccd)
         y = butler.get("x", ccd=ccd, immediate=True)
         self.assertEqual(bbox, y)
-        self.assert_(os.path.exists(
-            os.path.join(ROOT, "root", "out", "foo%d.pickle" % ccd)))
+        self.assertTrue(os.path.exists(
+            os.path.join(self.rootPath, "foo%d.pickle" % ccd)))
```

Comment:
I think this will fail if the current working dir is not the same as the directory above outpath?